pkg/pipelinerunmetrics/metrics.go

```diff
@@ -20,40 +20,67 @@ import (
 	"context"
 	"sync"
 
-	"github.com/tektoncd/chains/pkg/chains"
 	"go.opencensus.io/stats"
 	"go.opencensus.io/stats/view"
+	"go.opencensus.io/tag"
 	"knative.dev/pkg/logging"
 	"knative.dev/pkg/metrics"
+
+	common "github.com/tektoncd/chains/pkg/metrics"
+)
+
+const (
+	pipelineRunSignedName     common.Metric = "pipelinerun_sign_created_total"
+	pipelineRunSignedDesc     string        = "Total number of signed messages for pipelineruns"
+	pipelineRunUploadedName   common.Metric = "pipelinerun_payload_uploaded_total"
+	pipelineRunUploadedDesc   string        = "Total number of uploaded payloads for pipelineruns"
+	pipelineRunStoredName     common.Metric = "pipelinerun_payload_stored_total"
+	pipelineRunStoredDesc     string        = "Total number of stored payloads for pipelineruns"
+	pipelineRunMarkedName     common.Metric = "pipelinerun_marked_signed_total"
+	pipelineRunMarkedDesc     string        = "Total number of objects marked as signed for pipelineruns"
+	pipelineRunErrorCountName common.Metric = "pipelinerun_signing_failures_total"
+	pipelineRunErrorCountDesc string        = "Total number of PipelineRun signing failures"
```

Comment:
Since these are no longer in the `chains` package (they were only referenced in the `pipelinerunmetrics` package), they no longer need to be public, so the API is cleaned up a bit